var/spack/repos/builtin/packages/py-matplotlib/package.py

```diff
@@ -82,14 +83,16 @@ class PyMatplotlib(PythonPackage):
     depends_on('py-setuptools', type=('build', 'run'))  # See #3813
     depends_on('py-cycler@0.10:', type=('build', 'run'))
     depends_on('py-python-dateutil@2.1:', type=('build', 'run'))
+    depends_on('py-kiwisolver@1.0.1:', type=('build', 'run'), when='@2.2.0:2.2.4')
     depends_on('py-kiwisolver@1:', type=('build', 'run'), when='@2.2.0:')
```

Comment:
This kiwisolver business is confusing. `INSTALL.rst` has always said `>= 1.0.0`, but `setupext.py` and `lib/matplotlib.egg-info/requires.txt` have always said `>= 1.0.1`. To be on the safe side, I would remove the line you added above and use:
```suggestion
    depends_on('py-kiwisolver@1.0.1:', type=('build', 'run'), when='@2.2.0:')
```